dotnet/src/InternalUtilities/src/System/TypeConverterFactory.cs

```diff
@@ -16,7 +18,7 @@ internal static class TypeConverterFactory
     /// </summary>
     /// <param name="type">The Type of the object to convert.</param>
     /// <returns>A TypeConverter instance if a suitable converter is found, otherwise null.</returns>
-    internal static TypeConverter? GetTypeConverter(Type type)
+    internal static TypeConverter GetTypeConverter(Type type)
```

Comment:
Please add unit tests to the `KernelFunctionFromMethodTests1` class for testing object-to-JSON and JSON-to-object marshaling.

Reply:
I've added them, can you take a look?